schemas/field.json

```
@@ -193,7 +193,17 @@
                         },
                         "value": {
                             "description": "The value that the tag must have. (alternative to 'valueNot')",
-                            "type": "string",
+                            "type": {
+                                "anyOf": [
+                                    {
+                                        "type": "string"
+                                    },
+                                    {
+                                        "type": "array",
+                                        "items": { "type": "string" }
+                                    }
+                                ]
```

```suggestion
                                  {
                                      "type": "array",
                                      "items": { "type": "string" }
                                  }
```

Following https://github.com/ideditor/schema-builder/pull/87#issuecomment-2131163146, lets migrate the id-tagging-schema after this was released and merged over there.